fcc-test-suite-template-master/index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div class="container">
```

Comment:
- [ ] Good job so far 👍 
- [ ] Good using in-line comments 👏 
- [ ] I suggest you explain `why` you want the student to make changes so that they learn from it.
- [ ] As well please use emojis to sound more to the student ✌️ 
- [ ] Kindly apply the above mentioned to all the comments you made 🥂